Signal/src/AppDelegate.m

```diff
@@ -83,7 +83,7 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
     loggingIsEnabled = TRUE;
     [DebugLogger.sharedLogger enableTTYLogging];
 #elif RELEASE
-    loggingIsEnabled = Environment.preferences.loggingIsEnabled;
+    loggingIsEnabled = PropertyListPreferences.loggingIsEnabled;
```

Comment:
One more thought - what about avoiding the prod/dev discrepancy with file logging altogether? Are we really worried about people with debug builds not being able to figure out why logging isn't being persisted?

```
#ifdef DEBUG
    [DebugLogger.sharedLogger enableTTYLogging];
#end

loggingIsEnabled = PropertyListPreferences.loggingIsEnabled;
```